packages/widgetbook/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Unreleased
 
 - **FIX**: use `MapMixin` instead of `MapBase` for `KnobsRegistry`. ([#903](https://github.com/widgetbook/widgetbook/pull/903))
+- **FEAT**: Add Light Theme Support.
```

Comment:
The PR number should be included in the Changelog.

```suggestion
- **FEAT**: Add light theme support. ([#919](https://github.com/widgetbook/widgetbook/pull/919))
```

Reply:
Thanks @YoussefRaafatNasry. 
Also, thanks for helping me with the light theme colors.